app/src/main/java/org/mozilla/focus/home/HomeScreenBackground.java

```diff
@@ -64,12 +61,29 @@ void init() {
     @Override
     protected void onDraw(Canvas canvas) {
         super.onDraw(canvas);
-        canvas.drawRect(0, 0, getWidth(), getHeight(), paint);
+        if (!isNight) {
+            canvas.drawRect(0, 0, getWidth(), getHeight(), paint);
+        } else {
+            // Add status bar's height as a padding on top to let HomeFragment star background align with TabTrayFragment
+            setPadding(0, ViewUtils.getStatusBarHeight(((Activity) getContext())), 0, 0);
+        }
     }
 
     @Override
     public void onThemeChanged() {
         Drawable drawable = getContext().getTheme().getDrawable(R.drawable.bg_homescreen_color);
         setBackground(drawable);
     }
+
+    @Override
+    public void setNightMode(boolean isNight) {
+        super.setNightMode(isNight);
+        this.isNight = isNight;
+        if (this.isNight) {
+            setScaleType(ScaleType.FIT_START);
```

Comment:
We don't have to reset this in normal mode since the image resource will be removed in normal mode. ImageView's scale type definitely can be moved to xml.